src/lm_buddy/jobs/model_clients.py

```diff
@@ -44,10 +43,16 @@ class SummarizationPipelineModelClient(BaseModelClient):
     (model is loaded locally).
     """
 
-    def __init__(self, model: str, config: AutoModelConfig):
+    def __init__(self, model: str, config: HuggingFaceEvalJobConfig):
```

Comment:
Double check the type here here?

Reply:
Thanks for the heads up! Yep, the type is right: before I was not using the config as the pipeline just needed the model name, so I had decided to provide only the simplest config possible. Now I am passing the same config as with general HF models as I am gonna instantiate a tokenizer with that (it's even better as it makes the two calls consistent with each other)

```
        # depending on config, use the summarizer pipeline or directly call the model
        # for inference
        if config.evaluation.use_pipeline:
            logger.info(f"Using summarization pipeline. Model: {model_name}")
            model_client = SummarizationPipelineModelClient(model_name, config)
        else:
            logger.info(f"Using direct HF model invocation. Model: {model_name}")
            model_client = HuggingFaceModelClient(model_name, config)
```